x11-packages/languagetool-gui/build.sh

```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE=https://languagetool.org
+TERMUX_PKG_DESCRIPTION="GUI version of languagetool"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="$(
+	source "${TERMUX_SCRIPTDIR}/packages/languagetool-libs/build.sh"
+	echo "${TERMUX_PKG_VERSION}"
+)"
+TERMUX_PKG_SRCURL="$(
+	source "${TERMUX_SCRIPTDIR}/packages/languagetool-libs/build.sh"
+	echo "${TERMUX_PKG_SRCURL}"
+)"
+TERMUX_PKG_SHA256="$(
+	source "${TERMUX_SCRIPTDIR}/packages/languagetool-libs/build.sh"
+	echo "${TERMUX_PKG_SHA256}"
+)"
```

Comment:
I strongly disagree with this kind of sourcing version, URL, checksum. It breaks `./scripts/bin/revbump` and maybe more scripts.